src/tdd/isograma.ts

```diff
@@ -1,9 +1,21 @@
-function isograma(string) {
+function isograma(s: string): boolean {
 
-        if (string !== '-') {
-          callback(string);
-        }}
+    const buscaPalavra = s.length;
 
+    let lumberLetra = "L";
+    for (let i = s.length - 1; i >= 0; i--) {
+        lumberLetra = lumberLetra + s[i];
 
+        return true;
+    }
```

Comment:
Com isso aqui, o for loop roda uma vez e já retorna true para qualquer entrada que vc passar pra função (tirando string vazia). Acredito que essa parte não seja necessária, já que não é necessário inverter a string recebida pela função para identificar se ela é um isograma.